tools/harness-thci/OpenThread.py

```diff
@@ -2473,6 +2481,13 @@ def joinCommissioned(self, strPSKd='THREADJPAKETEST', waitTime=20):
             False: fail to start joiner
         """
         self.log("joinCommissioned on channel %s", self.getChannel())
+
+        if self.deviceRole in [
+                Thread_Device_Role.Leader,
+                Thread_Device_Role.Router,
+                Thread_Device_Role.REED,
+        ]:
+            self.__setRouterSelectionJitter(1)
```

Comment:
sgtm. Even though I think it's fine to setup `routerselectionjitter` for other roles.